R/label_roche.R

```diff
@@ -35,3 +35,58 @@ label_roche_pvalue <- function(big.mark = ifelse(decimal.mark == ",", " ", ","),
                                ...) {
   function(x) roche_pvalue(x, big.mark = big.mark, decimal.mark = decimal.mark, ...)
 }
+
+#' Round percentages
+#' @description
+#' This function only formats percentages between 0 and 1.
+#'
+#' @inheritParams gtsummary::style_percent
+#'
+#' @return A character vector of rounded percent values
+#' @name roche_percent
+#'
+#' @examples
+#' x <- c(0.0008, 0.9998)
+#'
+#' roche_percent(x)
+#' label_roche_percent()(x)
+NULL
+
+#' @export
+#' @rdname roche_percent
+roche_percent <- function(x,
+                          prefix = "",
+                          suffix = "",
+                          digits = 1,
+                          big.mark = ifelse(decimal.mark == ",", " ", ","),
+                          decimal.mark = getOption("OutDec"),
+                          ...) {
+  percent <- x * 100
```

Comment:
Instead of multiplying this here, can you add `scale=100` to the function calls?

Reply:
done!